.gitignore

```diff
@@ -1,3 +1,4 @@
 *.zip
 **/.zat
 **/tmp
+.DS_Store
```

Comment:
Usually this is ignored at the user folder level.

Reply:
Right, I was wondering why this wasn't included.